requirements/extras/huggingface_requirements.txt

```diff
@@ -1,2 +1,5 @@
 accelerate>=0.24.1,<=0.27.0
 sagemaker_schema_inference_artifacts>=0.0.5
+uvicorn>=0.30.1
```

Comment:
Are these needed? Doesn't look like we import them anywhere?

Reply:
they're imported in the app.py file